satellite/metainfo/validation.go

```diff
@@ -225,12 +230,56 @@ func (endpoint *Endpoint) validateBucket(ctx context.Context, bucket []byte) (er
 	if len(bucket) == 0 {
 		return errs.New("bucket not specified")
 	}
-	if bytes.ContainsAny(bucket, "/") {
-		return errs.New("bucket should not contain slash")
+
+	if len(bucket) < 3 || len(bucket) > 63 {
+		return errs.New("bucket name must be at least 3 and no more than 63 characters long")
```

Comment:
Shouldn't we use for this errors the `Error` class?

This apply to any other error returned in this function and unexported functions like `validateBucketLabel`

Reply:
you are right, done